### PR TITLE
add form for insertion of events client side

### PR DESCRIPTION
**Pull request by DaRavenox**
We want the user to be able to add calendar events through the client. Rudimentary bootstrap form along with the necessary framework to handle such an addition is included here.

### CODE REVIEW

File: dev/index.html [hunk 1]
```diff
@@ -22,13 +22,13 @@
 <body>
 
 	<!--<nav id="nav">
```
Comment (dskeppstedt) [inline dev/index.html]: Why is this, in this commit?

File: dev/js/authenticate.js [hunk 2]
```diff
@@ -80,12 +80,17 @@ let startApp = function() {
...
-		let day = frm.elements[1].value.substring(0,2);
-		let sDate = new Date(year,month,day,parseInt(frm.elements[2].value.substring(0,2)),parseInt(frm.elements[2].value.substring(2,4)));
-		let eDate = new Date(year,month,day,parseInt(frm.elements[3].value.substring(0,2)),parseInt(frm.elements[3].value.substring(2,4)));
+		for (var i = 0; i < frm.length; i++) {
+			console.log(frm.elements[i].value);
+		}
+		let year = parseInt(frm.elements[1].value.substring(0,4));
+		let month = parseInt(frm.elements[1].value.substring(5,7))-1; //Horrible datehack here as well. :)
+		let day = parseInt(frm.elements[1].value.substring(8,10));
+		console.log(year)
```
Comment (AndreasBrommund) [inline dev/js/authenticate.js]: Do we need this, not the best idea to print debug in prod

Comment (DaRavenox) [reply dev/js/authenticate.js]: Oh fuck me, you are right, will remove it.

Comment (AndreasBrommund) [reply dev/js/authenticate.js]: Looks good from my point of view, after your change

**Review by AndreasBrommund — APPROVED** [review]

**Comment by AndreasBrommund** [comment]
It's ok to merge for me, but talk to David before @DaRavenox

**Review by dskeppstedt — APPROVED** [review]
👍🏻

**Pull request merged**